temporal/api/cloud/auditlog/v1/message.proto

```diff
@@ -0,0 +1,107 @@
+syntax = "proto3";
+
+package temporal.api.cloud.auditlog.v1;
+
+option go_package = "go.temporal.io/api/cloud/auditlog/v1;auditlog";
+option java_package = "io.temporal.api.cloud.auditlog.v1";
+option java_multiple_files = true;
+option java_outer_classname = "MessageProto";
+option ruby_package = "Temporalio::Api::Cloud::AuditLog::V1";
+option csharp_namespace = "Temporalio.Api.Cloud.AuditLog.V1";
+
+import "google/protobuf/timestamp.proto";
+
+// LogRecord represents an audit log entry from Temporal, structured for easy parsing and analysis.
+message LogRecord {
+  // Time when the log was emitted from the source
+  google.protobuf.Timestamp emit_time = 1;
+
+  // Level of the log, i.e. info, warn, error etc
+  LogLevel level = 2;
+
+  // Operator email address
+  string user_email = 3;
+
+  // Operator IP address or server name
+  string caller_ip_address = 4;
+
+  // Operation performed
+  string operation = 5;
+
+  // Detailed information about the operation.
+  OperationDetail details = 6;
+
+  // Operation status from API call
+  string status = 7;
+
+  // Admin or System
+  LogCategory category = 8;
+
+  // Version of the log entry
+  int32 version = 9;
+
+  // Unique ID for the log record.
+  string log_id = 10;
```

Comment:
For other things that have an ID, like `User`, we don't prefix the type of ID if it applies to the proto message as a whole. May be best as just `id`.